src/bitmap/bitmap_8k.rs

```diff
@@ -0,0 +1,314 @@
+use crate::bitmap::sorted_u16_vec::SortedU16Vec;
+use crate::bitmap::store::Store;
+use std::borrow::Borrow;
+use std::ops::{BitAndAssign, BitOrAssign, BitXorAssign, RangeInclusive, SubAssign};
+
+pub const BITMAP_LENGTH: usize = 1024;
+
+#[derive(Clone)]
+pub struct Bitmap8K {
+    len: u64,
```

Comment:
Just to be sure, here we prefer storing an `u64` instead of an `u16` even if there can be only 65536 values in this container because converting from an `u16` to an `u64` when getting the length is not good for performance, right? And it doesn't that much to use an `u16` for alignment issues, it will always be 2 words on a 64bit machine.

Reply:
It's a u64 because `store::len` returned u64. Another case of "I didnt want to touch it"

Reply:
Yup, and I think that's better like that!